.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": true,
+  "trailingComma": "none",
+  "arrowParens": "always",
```

Comment:
Sometimes, I like to avoid having parenthesis in an arrow function if there's only a single argument. What is your opinion on this?

Reply:
I'm of the same opinion but the [recent update](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev) to Prettier stated this to be not a good dev experience, so I've added it.

Reply:
Also, what are your thoughts regarding `trailingComma`?

Reply:
In that case, I think  we can keep `"arrowParens": "always"` since it's a good dev experience. For trailingComma, It looks weird when there's a trailing comma, but since I was using eslint-config-google which enforces them, I had let them be but now it makes sense to keep it as `none`.